.circleci/config.yml

```diff
@@ -346,15 +346,6 @@ jobs:
       - run: git --no-pager diff
       - run: git --no-pager diff --quiet
 
-  docs-check:
-    executor: golang
-    steps:
-      - install-deps
-      - prepare
-      - run: make docsgen
-      - run: git --no-pager diff
-      - run: git --no-pager diff --quiet
```

Comment:
This is really useful, is there an equivalent of this on the new site?

Reply:
There will hopefully be soon. It's on my list to port it.